doc/tutorial/using_gpu.txt

```diff
@@ -485,11 +148,9 @@ the GPU object directly.  The following code is modifed to do just that.
   else:
       print('Used the gpu')
 
-Here the :func:`theano.gpuarray.basic_ops.GpuFromHost(None)` call
-means "copy input to the GPU", with ``None`` the default GPU context when not
-explicitly given. However during the optimization phase,
-since the result will already be on the gpu, it will be removed.  It is
-used here to tell theano that we want the result on the GPU.
+Here ``tensor.exp(x).transfer('None')`` means "copy ``exp(x)`` to the GPU",
+with ``None`` the default GPU context when not explicitly given.
+For information on how to set GPU contexts, see :ref:`tut_using_multi_gpu`. 
 
 The output is
 
```

Comment:
You would have to update the output below, it still says "Using device cuda0:" instead of "Mapped name None to device cuda0: ..."